ahiv/kafka/connection.h

```diff
@@ -78,54 +80,51 @@ class Connection : public uvw::Emitter<Connection> {
   }
 
  private:
-  void requestMetadataForTopicsWithRetry(std::vector<std::string>& wantedTopics, bool autoCreate, int8_t retries) {
-    ahiv::kafka::protocol::packet::MetadataRequestPacket requestPacket =
-        ahiv::kafka::protocol::packet::MetadataRequestPacket(
-            wantedTopics, autoCreate, false, false);
-
-    ahiv::kafka::protocol::Buffer buffer;
-    buffer.EnsureAllocated(requestPacket.Size());
-    requestPacket.Write(buffer);
-
-    this->SendToFirstConnection(buffer, [this, &buffer, &retries, &wantedTopics, autoCreate](
-        protocol::Buffer& responseBuffer) {
-      ahiv::kafka::protocol::packet::MetadataResponsePacket
-          metadataResponsePacket;
-      metadataResponsePacket.Read(responseBuffer);
-
-      for (const auto& broker : metadataResponsePacket.brokers) {
-        auto tcpConnection = this->consumeFromMetadata(broker);
-        if (tcpConnection != nullptr) {
-          this->connectionInfoByNodeId.insert(
-              std::make_pair(broker.nodeId, tcpConnection->connectionConfig));
-        }
-      }
+  void requestMetadataForTopicsWithRetry(std::vector<std::string>& wantedTopics,
```

Comment:
Parameter lists should be formatted in a way, that they either build vertical or horizontal rows. And boolean parameters are very hard to infer.
```suggestion
  void requestMetadataForTopicsWithRetry(
    const std::vector<std::string>& wantedTopics,
    CreationPolicy creationPolicy,
    int8_t retryCount) { ... }
    
```